mmcv/runner/hooks/logger/text.py

```diff
@@ -15,6 +16,8 @@ def __init__(self, interval=10, ignore_last=True, reset_flag=False):
     def before_run(self, runner):
         super(TextLoggerHook, self).before_run(runner)
         self.start_iter = runner.iter
+        self.json_log_path = '{}/{}.{}'.format(runner.work_dir,
+                                               runner.timestamp, 'log.json')
```

Comment:
It's more safe to use `osp.join(runner.work_dir, '{}.log.json'.format(runner.timestamp))`.